modules/microshift-otel-config-size.adoc

```diff
@@ -0,0 +1,241 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift_telemetry-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-config-size_{context}"]
+= Setting {microshift-short} Observability configurations
+
+The amount and complexity of the retrieved data depends on predefined configurations. These configurations determine the number of data sources and the amount of collected data that is transmitted. The configurations are defined as small, medium, and large (default).
+
+Replace the values of the `exporters.otlp.endpoint` and `services.telemetry.metrics.readers[0].endpoint` fields with the IP address or host name of the remote back end. This IP address resolves to the local node's host name. Any unreachable endpoint is reported in the {microshift-short} observability service logs.
+
+.Procedure
+
+The `opentelemtry-collector.yaml` file includes a minimal set of data for monitoring the system, node, and workload resource use for the CPU, memory, disk drive, and network. All warnings for cluster events are included in the collected data. This setting transmits the following data:
```

Comment:
I would remove the "minimal" because default is large, so it's not minimal :)
I actually removed that description from the file in ushift repo (https://github.com/openshift/microshift/pull/4903).

```suggestion
The `opentelemetry-collector.yaml` file includes a minimal set of data for monitoring the system, node, and workload resource use for the CPU, memory, disk drive, and network. All cluster events are included in the collected data. This setting transmits the following data:
```